tls/ttls/ssl_srv.c

```diff
@@ -30,24 +30,26 @@
 
 #if defined(MBEDTLS_SSL_SRV_C)
 
+#if defined(MBEDTLS_PLATFORM_C)
+#include "platform.h"
+#else
+#include <stdlib.h>
+#define mbedtls_calloc    calloc
+#define mbedtls_free      free
+#define mbedtls_time      time
+#define mbedtls_time_t    time_t
+#endif
```

Comment:
I think we should split (at least imaginary) mbedTLS code to two pieces: system dependent code and crypto algorithms implementation. The first piece is supposed to be reworked to be well integrated with Tempesta infrastructure, so we can just roughly rewrite system and libc calls (e.g. just replace `malloc()` by `kmalloc()` calls w/o intermediate `mbedtls_` defines). The second part is good to leave as is with as minimum changes as possible to be able to apply patches from further mbedTLS versions. However, the crypto routines should be considered to merge woth linux/crypto.

So in this case and few similar cases at the below (e.g for pthread calls), it seems the `mbedtls_` defines are unnecessary.